src/layouts/components/Editor/extensions/Image/Image.ts

```diff
@@ -80,6 +90,40 @@ export const IsomerImage = Image.extend({
         editor.state.selection.replace(tr)
         return true
       },
+
+      setImageStyle: (styleOptions) => ({ tr, dispatch, editor }) => {
+        const { from, to } = tr.selection
+
+        let style = ""
+
+        if (styleOptions.width) {
+          if (styleOptions.width < 1 || styleOptions.width > 100) {
+            console.error(
+              "Invalid width value. It should be between 1 and 100."
```

Comment:
just double checking here - do we have validation on the `form` for this range? 

we can sanitise + validate the input at point of entry and provide more feedback to user + safety to us

Reply:
Yes! As I mentioned [in this comment](https://github.com/isomerpages/isomercms-frontend/pull/1736#discussion_r1422241724), we already have the validation on the `form` UI to disable the save button when the value is not within the range of 1-100. 

This is just an extra layer of protection for cases like users using inspection mode to call the method directly that bypasses the UI. I'm inclined to keep it.

Reply:
i think if they inspect it's kinda in their own hands already but sure, no issues with keeping this!